https://issues.redhat.com/browse/ACM-7540  Improving obs doc from PM review (#5759)
open-cluster-management/rhacm-docs: observability/observability_support.adoc

[#observability-support]

> = Support

Reviewer: If you look across the doc, we don't have these stand-alone support topics. This seems odd to me. If this is a one off, we need to be **clearer in the title** because what is the difference between this and the actual section we have for troubleshooting or the other "support =-type" doc out there. It's sort of like the request for FAQ that we aren't doing because we don't want to start adding things to books that are not added across the doc.

I could name this Observability support, or Observability support information.

Reviewer 2: it is correct that Observability has a unique need for this support statement. Other areas of the product do not rely on any external API compatibility like this. The primary driver for these statements is the need for object storage.
In that case, I would expect perhaps OADP (backup and restore) might also have a similar need for such verbiage.

If we did find that there are multiple components that need this verbiage, I would be OK to have a common area that we can point to, but, I believe that is too much effort for this current review.

Author: Note to self: An issue should be created to discuss adding Support files for components that need them @swopebe @berenss

> = Support

Author: ```suggestion
= Observability support
```

- {product-title-short} is tested with and fully supported by Red Hat OpenShift Data Foundation, formerly {ocp}. 

> - {product-title-short} supports the function of the multicluster observability operator on user-provided third-party object storage that is S3 API compatible. The observability service uses Thanos supported, stable object stores.

Reviewer: Are we naming this in the doc the way we should? multicluster observability operator

(Let's look at the OperatorHub and see how it is named and go with that?)

Reviewer 2: MCO is not actually available in the OperatorHub. It is an operator that gets automatically bundled in with RHACM (which the RHACM operator itself is available in OperatorHub). I think the way it is written here is fine

Reviewer: I didn't find it. 

It's not written the same way in this doc, so _multicluster observability operator_ should be the same throughout the file, this will help with search and with future knowledge graphs.

Reviewer: I see `multicluster-observability-operator`. In other areas of the doc, you would see it like: Multicluster Observability operator (a title of a technology in the naming guidance). Something to consider...it helps if I can see what the user sees.

Author: i monospace multicluster-observability-operator when i refer to the pod, or attempt to be consistent with it. The resource name is `MultiClusterObservability`

> - {product-title-short} use commercial, reasonable efforts to assist in the identification of the root cause. If a support ticket is raised and the root cause has been determined to be a result of the customer-provided S3 compatible object storage, then the issue must be resolved using the customer support channels.

Reviewer: {product-title-short} use commercial, reasonable

add s to use and what is "reasonable" (again, this seems bloggy or article-y.)

Reviewer 2: it's business speak, certainly feel you there. When forming this statement, we largely used verbiage that has been leveraged for the past few decades in IBM and other parts of Red Hat.

Reviewer: I think we need to work on this a bit.

- {product-title-short} support efforts include reasonable efforts to identify root causes. If you open a support ticket and the root cause is the customer-provided S3 compatible object storage, then you must open an issue using the customer support channels.

And there should early in this doc be a simple link to support, which I think we already have in our support docs.

This would not be so odd if it were here: https://github.com/stolostron/rhacm-docs/blob/2.9_stage/release_notes/release_notes.adoc

But if you feel it is only for this section, I am fine with a little editing.

Author: Updated this

> - {product-title-short} use commercial, reasonable efforts to assist in the identification of the root cause. If a support ticket is raised and the root cause has been determined to be a result of the customer-provided S3 compatible object storage, then the issue must be resolved using the customer support channels.

Author: adding Brandi's suggestions
```suggestion
- {product-title-short} support efforts include reasonable efforts to identify root causes. If you open a support ticket and the root cause is the customer-provided S3 compatible object storage, then you must open an issue using the customer support channels.
```

> - {product-title-short} does not commit to fix support tickets raised by customers, where the root cause identified is the S3 compatible object storage provider.

Author: ```suggestion
- {product-title-short} does not support fixing support tickets raised by customers, where the root cause identified is the S3 compatible object storage provider.
```

> - {product-title-short} does not commit to fix support tickets raised by customers, where the root cause identified is the S3 compatible object storage provider.

Reviewer: It seems very odd that we have this statement in our doc. First of all, we don't want to say the product or product team is committing or not committing to something in a doc. That should be in the matrix or in an article. @berenss this is odd to me, is there a better place for this?

Reviewer 2: Thank you - the BU has asked that this remains in the DOC portal as the defacto standard for supported functions and capabilities. In fact the topic has come up many times via customers and partners that leverage this support statement for a crisp understanding of what role Red Hat support can be involved in, and their answer was always met with kind reassurance that we document it within the product manuals.

Reviewer: @berenss I think we also need to make sure that it stays within the standards that we keep in tech writing for Red Hat, which is not usually known by development or PM, but should be known by all writers and CCS folks.

So we should not say _The product commits_ to something. We should say:

RHACM does _not support S3 compatible object storage provider_ (or something like that). We don't refer to "the customer" in the official doc, so it is unusual to have this here.

I am fine with most of this as long as we edit it a bit. 

Also, why would this be buried in the Observability doc if it applies to the larger product? I feel it should honestly go in the _Troubleshooting section_ or _Release notes_ if it is in fact about RHACM support, no?
